server/views/pages/prisonIncentiveLevelForm.njk

```diff
@@ -0,0 +1,135 @@
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {% include "../partials/messages.njk" %}
+      {% include "../partials/formErrorSummary.njk" %}
+
+      <h1 class="govuk-heading-xl">{{ prisonIncentiveLevel.levelDescription }} level in {{ prisonName }}</h1>
+
+      {{ govukBackLink({
+        text: "Back",
+        href: "/prison-incentive-levels/view/" + prisonIncentiveLevel.levelCode
+      }) }}
+
+      <form id="form-{{ form.formId }}" method="post" novalidate>
+
+        {% call govukFieldset({
+          legend: {
+            text: "Spend limits",
+            classes: "govuk-fieldset__legend--m"
+          }
+        }) %}
+
+          {% set formId = "remandTransferLimit" %}
+          {% set field = form.getField(formId) %}
+          {{ govukInput({
+            label: { text: "Remand transfer limit" },
+            prefix: { text: "£" },
+            classes: "govuk-input--width-5",
+            id: form.formId + "-" + formId,
+            name: formId,
+            value: field.value,
+            errorMessage: { text: field.error } if field.error else undefined
+          }) }}
+
+          {% set formId = "remandSpendLimit" %}
+          {% set field = form.getField(formId) %}
+          {{ govukInput({
+            label: { text: "Remand spend limit" },
+            prefix: { text: "£" },
+            classes: "govuk-input--width-5",
+            id: form.formId + "-" + formId,
+            name: formId,
+            value: field.value,
+            errorMessage: { text: field.error } if field.error else undefined
+          }) }}
+
+          {% set formId = "convictedTransferLimit" %}
+          {% set field = form.getField(formId) %}
+          {{ govukInput({
+            label: { text: "Convicted transfer limit" },
+            prefix: { text: "£" },
+            classes: "govuk-input--width-5",
+            id: form.formId + "-" + formId,
+            name: formId,
+            value: field.value,
+            errorMessage: { text: field.error } if field.error else undefined
+          }) }}
+
+          {% set formId = "convictedSpendLimit" %}
+          {% set field = form.getField(formId) %}
+          {{ govukInput({
+            label: { text: "Convicted spend limit" },
+            prefix: { text: "£" },
+            classes: "govuk-input--width-5",
+            id: form.formId + "-" + formId,
+            name: formId,
+            value: field.value,
+            errorMessage: { text: field.error } if field.error else undefined
+          }) }}
+
+        {% endcall %}
+
+        {% call govukFieldset({
+          legend: {
+            text: "Visit orders",
+            classes: "govuk-fieldset__legend--m"
+          }
+        }) %}
+
+          {% set formId = "visitOrders" %}
+          {% set field = form.getField(formId) %}
+          {{ govukInput({
```

Comment:
We could use `inputmode: "numeric"` for the visits, as suggested here: https://design-system.service.gov.uk/components/text-input

Reply:
good idea